src/VisualStudio/LiveShare/Impl/Client/RemoteLanguageServiceWorkspace.cs

```diff
@@ -408,7 +455,10 @@ protected override void ApplyDocumentTextChanged(DocumentId documentId, SourceTe
                         return textContainer.TryGetTextBuffer();
                     });
 
-                    UpdateText(textBuffer, text);
+                    if (textBuffer != null)
```

Comment:
When can this happen? Do we have some bug if it _doesn't_ happen?

Reply:
(At least before any bug would make itself clearly known with a nice crash or error message.)